src/styles/block/button-buy.scss

```
@@ -0,0 +1,23 @@
+.buy {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 164px;
```

No need to hardcode width
```suggestion
  width: 100%;
```